cumulus/polkadot-parachain/polkadot-parachain-lib/src/fake_runtime_api/utils.rs

```diff
@@ -202,7 +202,7 @@ macro_rules! impl_node_runtime_apis {
 
 				fn dispatch_benchmark(
 					_: frame_benchmarking::BenchmarkConfig
-				) -> Result<Vec<frame_benchmarking::BenchmarkBatch>, sp_runtime::RuntimeString> {
+				) -> Result<Vec<frame_benchmarking::BenchmarkBatch>, alloc::string::String> {
```

Comment:
```suggestion
				) -> Result<Vec<frame_benchmarking::BenchmarkBatch>, String> {
```

This code is never compiled for `no_std`.

Reply:
It actually is used like that in CI, had to revert this suggestion